src/Tools/SchemaTool.php

```diff
@@ -282,7 +286,17 @@ public function getSchemaFromMetadata(array $classes): Schema
                     }
 
                     if ($pkColumns !== []) {
-                        $table->setPrimaryKey($pkColumns);
+                        if (class_exists(PrimaryKeyConstraint::class)) {
```

Comment:
Can you factor this method out? its repeated three times.

Reply:
done, not sure though how I can satisfy PHPStan